tests/gdx-tests-android/assets/data/maps/tiled-atlas-processed/test.tmx

```diff
@@ -10,9 +10,11 @@
    eJxjYBgFo2D4AC5GBgZuKmAeRoh5ACn1AOU=
   </data>
  </layer>
- <layer height="10" name="Tile Layer 2" width="20">
-  <data compression="zlib" encoding="base64">
-   eJzlzjkOggAQBVBqFLF2r13Q2r12wxvoDfQI3ACP7CuJ0ZDQEX/ymsnM5AfBfyWkQZOIFjHtiv86dOnRZ8CQUcld9mM+ZsKUGQlzFhX7LVmxZsOWHXtu3Au7+RefOXDkxJkLKVcePEv6vArqkDdo6gzB
-  </data>
- </layer>
-</map>
\ No newline at end of file
+ <group name="testGroup">
```

Comment:
it would be nice to have a separated test case with more complete "group" test like : nesting several groups, with different offsets to ensure offsets are recursively applied and so on. See https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#group